gsoc/admin.py

```diff
@@ -353,10 +354,10 @@ class HiddenUserProfileAdmin(admin.ModelAdmin):
         "hidden",
         "reminder_disabled",
         "current_blog_count",
-        "role", 
+        "role",
         "gsoc_invited",
     )
-    list_filter = ("hidden", "reminder_disabled", "role", "gsoc_invited")
+    list_filter = ("hidden", "reminder_disabled", "role", "gsoc_invited", GSoCYearFilter)
```

Comment:
As I mentioned, why have a new class? just display the data that is already there.